### DISCUSSION TITLE
Upload to folder via api

### DISCUSSION THREAD

**Discussion by charonfaustinus** [diced/zipline#591] [Ideas]
I use zipline with a different application other than ShareX. And I want to upload directly into the folder prior to the application I've set it up using CURL command. Is there a way to do it using API? Or this just not implemented yet.

**Comment by diced** [comment]
you can use the `x-zipline-folder` header set to the ID of any folder

it is outlined in the docs: https://zipline.diced.sh/docs/guides/upload-options
![image](https://github.com/user-attachments/assets/dda44265-0ee2-4eef-9fe7-7c8471eae605)
